browser/src/UI/components/BufferScrollBar.tsx

```diff
@@ -72,12 +72,12 @@ export class BufferScrollBar extends React.PureComponent<IBufferScrollBarProps,
                 width: "100%",
             }
 
-            return <div style={markerStyle} key={m.line.toString() + m.color}/>
+            return <div style={markerStyle} key={`${m.line}-${m.color}-${i}-${top}`}/>
```

Comment:
I'm not sure if this is actually the correct fix for this - this seems like it could cause aggressive re-paints for the buffer scroll bar (which has caused performance issues). I think we may want to try putting a key on the scroll bar itself (or even the `ActiveWindow`). Could we track this in a separate issue / PR?